masterlist.yaml

```diff
@@ -25043,13 +25052,10 @@ plugins:
     global_priority: 105
   - name: 'Skyrim Unlocked.esp'
     global_priority: 70
-    inc:
-      - name: "Open Cities Skyrim.esp"
-        condition: 'not active("Open Cities Skyrim - Skyrim Unlocked Patch.esp")'
     msg:
-      - type: warn
-        condition: 'active("Open Cities Skyrim.esp") and not active("Open Cities Skyrim - Skyrim Unlocked Patch.esp")'
-        content: 'Incompatible with: Open Cities Skyrim. You must install the compatibility patch.'
+      - <<: *reqPatchForX
```

Comment:
This pattern is why I think loot/loot.github.io#55 would be good to have... Given `inc`'s current limitations though, this is fine.